File: Data Structures & Algorithms/remove-element/submission-1.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int k = nums.size() - std::count(nums.begin(), nums.end(), val); 
        nums.erase(std::remove(nums.begin(), nums.end(), val), nums.end());

        return k;
    }
};

File: Data Structures & Algorithms/3sum-smaller/submission-1.py
class Solution:
    def threeSumSmaller(self, nums: list[int], target: int) -> int:
        ns = sorted(nums)
        count = 0
        i = 0
        for j in range(i + 1, len(ns)):
            for k in range(j + 1, len(ns)):
                if ns[i] + ns[j] + ns[k] < target:
                    count += 1
            i += 1
        return count
